pages/juang961225/style.css

```diff
@@ -0,0 +1,98 @@
+body {
+    display: flex;
+    flex-wrap: nowrap;
+    margin: 0px;
+}
+.main {
```

Comment:
`main` could not be the best way to name the block... this does not refer to its functionality.

Could you try with another name like `slider` or `carousel`?

Reply:
resolved, change the name 'main' to 'slider